### DISCUSSION TITLE
Not doing well on hindhi language?

### DISCUSSION THREAD

**Discussion by akashAD98** [openai/whisper#118] [Q&A] [answered]
It's detecting audio as Hindhi, but the result decoded audio showing in urdu language, whats wrong with this?

<img width="660" alt="image" src="https://user-images.githubusercontent.com/62583018/192129967-661106d5-124f-4dbe-9bc9-3cdc8aa563da.png">

**Comment by akashAD98** [comment, marked as answer]
i tried base model & accuracy is not good, trying medium & large model accuracy is very good

**Reply by Fayzan-Bhatti** [reply]
There are several potential reasons why the model you are using might be producing incorrect transcriptions or translations, including: the language and domain of the audio, the quality of the model, and the input format. To improve the accuracy of the model, it is important to consider these factors and try to identify the root cause of any issues. You may also want to try using a different model (medium or large) or adjusting the input format to see if that improves performance.

**Comment by jongwook** [comment]
<img width="303" alt="image" src="https://user-images.githubusercontent.com/266841/192136944-e19fb7d7-9233-4469-897b-7948d87d3b14.png">

This is Table 11 from the paper; showing medium/large models perform significantly better than the base model, as you noticed.

**Comment by GokulNC** [comment]
Interesting that the model understood that Hindi & Urdu is the same language when spoken!
https://en.wikipedia.org/wiki/Hindustani_language

On Fleurs for Whisper-base, Urdu has a WER of 52, whereas Hindi is at 101.
Which answers the confusion :)

**Comment by pruthvishetty** [comment]
It's transcribing other Indian languages (ex: Kannada) in Urdu as well. Any ideas on how to fix this?
